src/App.tsx

```diff
@@ -117,7 +118,11 @@ export function App() {
 
   const isMobile = useMediaQuery(theme.breakpoints.down('desktop'));
   const [isAuthenticated, setIsAuthenticated] = useState(getCookie('userToken') !== 'undefined');
-  const getRole = localStorage.getItem('userRole') === 'undefined' ? null : localStorage.getItem('userRole') as RoleType | null;
+  // TODO: set role based on info from API, when userToken cookie is set
+  const getRole =
+    localStorage.getItem('userRole') === undefined
+      ? null
+      : (localStorage.getItem('userRole') as RoleType | null);
```

Comment:
co to za zmiana? wygląda jakby rebase coś cofnął

Reply:
Fakt, powinno być `'undefined'`